.azure/templates/build-test.yml

```diff
@@ -88,13 +88,13 @@ steps:
       set -e -x
       sudo apt-get install libacl1-dev libncurses5-dev pkg-config
       git clone --single-branch \
-                --branch "${ICEORYX_BRANCH:-master}" \
+                --branch "${ICEORYX_BRANCH:-v1.0.1}" \
                 "${ICEORYX_REPOSITORY:-https://github.com/eclipse-iceoryx/iceoryx.git}" \
                 iceoryx
       mkdir iceoryx/build
       cd iceoryx/build
       cmake -DCMAKE_BUILD_TYPE=${BUILD_TYPE} \
-            -DCMAKE_BUILD_SHARED_LIBS=on \
+            -DBUILD_SHARED_LIBS=on \
             -DCMAKE_INSTALL_PREFIX=install \
             -DROUDI_ENVIRONMENT=on \
```

Comment:
We should also check whether Iceoryx was built with `-DROUDI_ENVIRONMENT=ON`: that's the bit that produces the file (and turns out to have been the reason for my not having the testing package locally).